var/spack/repos/builtin/packages/py-pyfr/package.py

```diff
@@ -44,8 +50,9 @@ class PyPyfr(PythonPackage):
     depends_on("py-pytools@2016.2.1:", type=('build', 'run'))
     depends_on("py-scipy", type=('build', 'run'))
```

Comment:
scipy isn't listed as a dep for either version, should this be removed from Spack's `package.py` or added to pyfr's `setup.py`?

Reply:
Thanks, as mentioned in #28847, this is used opportunistically, and is recommended but not required.

Reply:
Can you add a comment above this line so I don't keep asking you lol?

Reply:
> Can you add a comment above this line so I don't keep asking you lol?

Yes of course. Thanks for the inputs here. I will update with the changes.

Reply:
Made it optional as a variant. Thanks